Dockerfile

```diff
@@ -26,6 +25,6 @@ RUN chmod a+x /app/scripts/*
 
 EXPOSE 8080
 
-USER ${USER_ID}
+USER ${USER}
 
-CMD /app/scripts/run.sh
\ No newline at end of file
+CMD ["sh", "/app/scripts/run.sh"]
```

Comment:
Can we add a new line to the bottom of this?

Reply:
Yep done